src/waltz/tls/fd_tls.h

```diff
@@ -230,7 +219,7 @@ fd_tls_sign( fd_tls_sign_t const * sign,
    across multiple TLS handshakes. */
 
 struct fd_tls {
-  fd_tls_rand_t       rand;
+  fd_tls_rand_t       rand;  /* may be NULL */
```

Comment:
@0x0ece recommended against nullable function pointers at runtime here. I still have to think about the most robust fix. We should probably have an `fd_tls_init` that fills out defaults and errors if any mandatory field is left uninitialized.